docs/functions.md

```diff
@@ -0,0 +1,73 @@
+# Function Introduction
+
+## Relating with VCS tools
+
+Cyclone can relate with various common VCS tools, such as git, svn, etc. After OAuth, it can pull codes from repository and create webhook. Whenever the user commit, submit a pull request or release a version to the repository, the webhook will trigger the CI/CD workflow. 
+
+- Creating a service relating with the VCS tools
+
+<div align="center">
+	<img src="./create_service.png" alt="create_service" width="500">
+</div>
+
+- Listing all services
+
+<div align="center">
+	<img src="./list_services.png" alt="list_services" width="500">
+</div>
+
+## Continus integration and security scanning
+
+All of the processes in workflow are visible. "prebuild" compile the executable files. "build" build the published image. "integration" execute the integrated test. "publish" publish the image and scan the vulnerabilities. "post build" do some relating operations after image published. "deploy" use the published image deploy a application to Kubernetes or any other container cloud platform. Cyclone can send email to notify the result of the workflow. All of the processes are shipped by container. It will wipe off the differences cause by environment. 
+
+- Logs of the workflow
+
+<div align="center">
+	<img src="./logs.png" alt="logs" width="500">
+</div>
+
+- Security scanning
+
+<div align="center">
+	<img src="./security.png" alt="security" width="500">
+</div>
+
+- Notifying via email 
+
+<div align="center">
+	<img src="./pagging.png" alt="pagging" width="500">
```

Comment:
@xiaomuqiao has deleted the color from the log. Because it will interfere the display on web.